R/utilities-entity.R

```diff
@@ -24,13 +24,13 @@ CompareBy <- enum(c(
 #' sim <- loadSimulation(simPath)
 #'
 #' parameters <- c(
-#'   getParameter(toPathString(c("Organism", "Liver", "Pericentral", "Volume")), sim),
-#'   getParameter(toPathString(c("Organism", "Liver", "Pericentral", "Volume")), sim),
-#'   getParameter(toPathString(c("Organism", "Liver", "Pericentral", "Weight (tissue)")), sim)
+#'   getParameter(toPathString(c("Organism", "Liver", "Volume")), sim),

```

Comment:
I had some pain here figuring out why the example would not run! The simple simulation is just too simple :)